ports/stm32f4/boards/openocd_stm32f4.cfg

```diff
@@ -0,0 +1,44 @@
+# This script configures OpenOCD for use with an ST-Link V2 programmer/debugger
+# and an STM32F4 target microcontroller.
+#
+# To flash your firmware:
+#
+#    $ openocd -f openocd_stm32f4.cfg \
+#        -c "stm_flash build-BOARD/firmware0.bin 0x08000000 build-BOARD/firmware1.bin 0x08020000"
+#
+# For a gdb server on port 3333:
+#
+#    $ openocd -f openocd_stm32f4.cfg
```

Comment:
Does this file have a license? It looks like it is copied from somewhere.

Reply:
It's from Micropython - it doesn't have anything, as per the linker files. Do you want me to put one in there?

Reply:
Nah, it's from micropython then it is fine.